charts/opentelemetry-demo/values.yaml

```diff
@@ -286,6 +291,10 @@ components:
         port: 8081
       initialDelaySeconds: 30
       periodSeconds: 10
+    initContainers:
+    - name: featureflagservice-init
+      image: busybox:latest
+      command: ['sh', '-c', 'until nc -z {{ include "otel-demo.name" . }}-ffspostgres 5432; do echo waiting for postgres; sleep 2; done']
```

Comment:
If postgres never comes up is this deployment stuck indefinitely?

Reply:
yes, but the featureflag service would never come up successfully if postgres never came up, so i don't think that's really a problem.